### ISSUE TITLE
magic square for 4x4 doesn't work under 1000 steps

### ISSUE THREAD

**Issue by Horopter** [python-constraint/python-constraint#43]
I've tried 4x4 magic square but it always returns None, even with MinConflictsSolver.

**Comment by Horopter** [comment]
Also may I have some more documentation about constraint propagation via doArc8? I don't understand that I am afraid.

**Comment by fjwillemsen** [comment]
@Horopter Yeah I was expecting as much 😂 Hope you didn't fail your assignment because of this! I took over this project two years ago so was going through old issues.

**Comment by Horopter** [comment]
@fjwillemsen I had to make my own subset of constraint lib to get an A+. ☺️ I ain't an expert but if you need a hand, I can contribute to this repo. Keep in touch.

**Comment by fjwillemsen** [comment]
That's very nice, glad to hear it - hope to see you around!
